tests/lib/rules/id-length.js

```diff
@@ -35,6 +35,7 @@ ruleTester.run("id-length", rule, {
         "var xyz = new ΣΣ();",
         "unrelatedExpressionThatNeedsToBeIgnored();",
         "var obj = { 'a': 1, bc: 2 }; obj.tk = obj.a;",
+        "var query = location.query.q || '';",
```

Comment:
I don't think we need a ternary-specific test but I can add if you feel strongly about it.